src/query/storages/index/benches/build_from_block.rs

```diff
@@ -65,10 +67,11 @@ fn bench_string(c: &mut Criterion) {
     let column = block.try_column_by_name("a").unwrap();
 
     let mut builder = Xor8Builder::create();
-    builder.add_keys(&column.to_values());
+    let values: Vec<DataValue> = (0..column.len()).map(|i| column.get(i)).collect();
+    builder.add_keys(&values);
```

Comment:
We should use for loop to `add_key`, to avoid using `collect`, or the bench is slow since there are too many value copies.